runtime/permissions.rs

```diff
@@ -1184,7 +1184,13 @@ impl Permissions {
         .as_ref()
         .map(|v| {
           v.iter()
-            .map(|x| NetDescriptor::from_string(x.clone()))
```

Comment:
I feel like instead we should change `NetDescriptor` to return a `Result<Self, ..>` then surface these errors to the user?

Reply:
@dsherret done